src/scripts/SequenceTrack.js

```diff
@@ -433,6 +427,15 @@ const SequenceTrack = (HGC, ...args) => {
     }
 
     drawTextSequence(tileX, tileWidth, tile) {
```

Comment:
Is there a reason why you moved this here from `initTile`? It will be executed a lot more often now.

Reply:
Yes, because initTile only gets called when the data is loaded. So when the options are changed, the text sequence is not redrawn. This causes a bug where if you try to change the font color in the options, nothing happens. Moving `drawTextSequence` fixes that bug.

Reply:
That makes sense. For some reason I thought a `rerender` would re-initialize the tiles.